components/global/Header/Navigation.js

```diff
@@ -42,15 +38,17 @@ export default function Navigation({
     }
     if (navList?.current?.contains(e.target)) return;
     setActive(null);
+    if (desktopSetter) desktopSetter(false);
   }
 
   function handleToggleClick(id) {
     setActive((prevActive) => (prevActive === id ? null : id));
+    if (desktopSetter) desktopSetter(true);
   }
 
   function handleSignOut() {
     setActive(null);
-    mobileSetter(false);
+    if (mobileSetter) mobileSetter(false);
```

Comment:
`mobileSetter` only exists in the mobile instance of the nav, and while it may be the case that here it always exists because of when `handleSignOut` is called, elsewhere I saw `mobileSetter` called conditional on its own existence, so decided to do the same here. elsewhere I saw